itl/command-plugins.conf

```
@@ -1780,6 +1785,96 @@ object CheckCommand "snmp" {
 			set_if = "$snmp_perf_oids$"
 			description = "Label performance data with OIDs instead of --label's"
 		}
+		// SNMP v2c options
+		"-C" = {
+			value = "$snmp_community$"
+			description = "Optional community string for SNMP communication (default is 'public')"
+			set_if = {{
+				if ((typeof(snmpversion) == Number && snmpversion <= 2) || (typeof(snmpversion) == String && (snmpversion == "1" || snmpversion == "2") || snmpversion == "2c")) {
```

```suggestion
				if ((typeof(snmpversion) == Number && snmpversion <= 2) || (typeof(snmpversion) == String && (snmpversion == "1" || snmpversion == "2" || snmpversion == "2c"))) {
```

The `&&` has a precedence over `||`.